tests/pom.xml

```diff
@@ -54,10 +83,32 @@
             <includes>
               <include>%regex[.*IntegrationTest.*]</include>
             </includes>
+            <systemPropertyVariables>
+              <app.deploy.port>${local.app.port}</app.deploy.port>
+            </systemPropertyVariables>
+          </configuration>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <version>2.19.1</version>
+          <configuration>
+            <excludes>
+              <exclude>**/*IntegrationTest.java</exclude>
+            </excludes>
           </configuration>
         </plugin>
       </plugins>
     </pluginManagement>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-deploy-plugin</artifactId>

```

Comment:
keeps the war/docker file from ever being deployed into oss which is configured in the tlp